src/styles/main.scss

```
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
+@import "./variables";
+@import "./calendar";
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
```

```suggestion
* {
```
Just set the font for the whole page or body and inherit where needed